docs/getting-started.md

```diff
@@ -34,221 +37,379 @@ The high level data flow is as follows:
+
+#### 4.2.4 Syncing with the Testnet
+
+If you want to sync with the testnet instead of the mainnet, just make sure the local node is a testnet node, or that you are hitting the Infura testnet endpoints. 
+
+## 5 Query the local Graph Node
+With the subgraph deployed to the locally running Graph Node, visit http://127.0.0.1:8000/ to open up a [Graphiql](https://github.com/graphql/graphiql) interface where you can explore the deployed GraphQL API for the subgraph by issuing queries and viewing the schema. 
+
+We have provided a few simple examples below, but please see the [Query API](graphql-api.md#1-queries) for a complete reference on how to query the subgraph's entities.
 
 #### Example
 Query all `Token` entities:
 ```graphql
-query {
+{
   tokens {
     id
-    owner
+    currentOwner
+  }
+}
+```
+Notice that it is just simply `tokens` pluralized, and it will return all of the entites in The Graph. 
+
+Later, when you've deployed the subgraph with this entity, you'll be able to query for a specific value, such as the token ID
+
+```graphql
+{
+  token(id: "c2dac230ed4ced84ad0ca5dfb3ff8592d59cef7ff2983450113d74a47a12") {
+    currentOwner
   }
 }
 ```
 
-## 6 Buidl 🚀
-Start building world-changing dApps on top of your newly deployed GraphQL interface 🗿✨.
+You can also organize the queries. This query below would organize all tokens by their ID, and return the current owner of each token. 
+
+```graphql
+{
+  tokens(orderBy: id) {
+    currentOwner
+  }
+}
+```
+
+GraphQL has a ton of functionality! Once again, check out the [Query API](graphql-api.md#1-queries) to figure out how to use all query functionalities. 
+
+## 6 Changing the schema, mappings, and manifest, and launching a new Subgraph
+
+When you first start building the subgraph, it is likely you will make a few iterations to the manifest, mappings, or schema. If you update any of them, you should run `yarn codegen` and `yarn deploy`. This will post the new files on ipfs and deploy the new subgraph. Note that the graph node can track multiple subgraphs, so you can do this as many times as you like.
+
+## 7 Example Subgraphs You can look to 
+
+Here is a list of current subgraphs that we have open sourced, which should help you build your own subgraph:
+* https://github.com/graphprotocol/ens-subgraph
+* https://github.com/graphprotocol/decentraland-subgraph (most complex)
+* https://github.com/graphprotocol/adchain-subgraph
+* https://github.com/graphprotocol/memefactory-subgraph (only for testing, not live on Ethereum mainnet)
+
+It might be worth your while to clone one of them, and start the project from there. 
```

Comment:
Not a good idea, it could lead to disqualification at hackathons, because hackers are only allowed to start coding after the hackathon has kicked off. Cloning existing repos will lead to commits that predate the event.

Reply:
yeah I was writing it with the thought of any person coming here, and not just the hackathon. I will just mention that these are our repositories they can refer to